src/db.rs

```diff
@@ -332,7 +332,11 @@ CREATE table review_prefs (
     assigned_prs INT[] NOT NULL DEFAULT array[]::INT[]
 );",
     "
-CREATE EXTENSION intarray;
-CREATE UNIQUE INDEX review_prefs_user_id ON review_prefs(user_id);
+CREATE EXTENSION IF NOT EXISTS intarray;",
+    "
+CREATE UNIQUE INDEX IF NOT EXISTS review_prefs_user_id ON review_prefs(user_id);
  ",
+    "
+ALTER TABLE review_prefs ADD COLUMN IF NOT EXISTS max_assigned_prs INTEGER DEFAULT NULL;
+",
```

Comment:
The migration should now be correctly formatted.

Adding the clause `IF NOT EXISTS` should address the concern addressed in [this comment](https://github.com/rust-lang/triagebot/pull/1792/commits/70dd15e02a30932af4691f5fc0f020c273faeab6#r1567313497) about re-running them.